src/components/buttons/PopoverButton/PopoverButton.tsx

```diff
@@ -82,15 +84,20 @@ export default function PopoverButton(props: PopoverButtonProps) {
       style={{
         textTransform: 'none',
       }}
+      disabled={isDisabled}
+      aria-disabled={isDisabled}
     >
       {buttonDisplayContent}
     </Button>
   );
 
   return (
-    <>
+    <div style={{
+      width: 'fit-content', 
+      ...(isDisabled ? { cursor: 'not-allowed' } : {})
+    }}>
       {button}
       {menu}
-    </>
+    </div>
```

Comment:
Completely unnecessary comment: if you ever wanted to add a tooltip for free you can add a `title` attribute to this div.

![mustard](https://user-images.githubusercontent.com/24446573/223855995-359fe23b-e29c-43f1-af3d-81d80ad7738c.gif)

The title attribute specifies extra information about an element which is revealed on hover. The title attribute is one of the "global attributes" that is available on all elements in the DOM.

Reply:
That's the first time anybody has called my words sage.

Reply:
Oh, right! 🧙 I get that one a lot. 🪄 ⚡ 🥔